quizeek/src/server-actions/user/auth.ts

```diff
@@ -0,0 +1,7 @@
+'use server';
+
+import { signOut } from '@/auth';
+
+export const signOutAction = async (): Promise<void> => {
```

Comment:
You did not alter the files in `server-actions/user/index.ts` and `server-actions/index.ts`

Reply:
That action is not being used at the moment, I think we could get rid of it.